app/controllers/admin/videos_controller.rb

```diff
@@ -1,18 +1,24 @@
 class Admin::VideosController < Admin::BaseController
   def edit
-    @video = Video.find(params[:video_id])
+    @video = Video.find(params[:id])
   end
 
   def update
-    video = Video.find(params[:id])
-    video.update(video_params)
+    @video = Video.find(params[:id])
+    if @video.update(video_params)
+      flash[:success] = 'Video Updated!'
+      redirect_to tutorial_path(@video.tutorial_id)
+    else
+      flash[:error] = @video.errors.full_messages.to_sentence
+      render :edit
+    end
```

Comment:
Added this action because it was never actually set up and was causing our coverage to be low

Reply:
awesome